tests/cnf/core/network/security/tests/nftables-custom-firewall-test.go

```diff
@@ -91,62 +96,135 @@ var _ = Describe("nftables", Ordered, Label(tsparams.LabelNftablesTestCases), Co
 
+				By("Verify that ingress TCP port 8088 is not blocked")
+				err = cmd.ValidateTCPTraffic(masterPod, ipv4NodeAddrList, interfaceNameNet1, frrconfig.ContainerName,
+					portNum8088)
+				Expect(err).ToNot(HaveOccurred(),
+					"Failed to send ingress TCP traffic over port 8088")
+			})
+
+		It("Verify a custom firewall nftables is reloaded after host reboot with all existing rules",
+			reportxml.ID("77144"), func() {
+				By("Verify ICMP connectivity between the master Pod and the test pods on the workers")
+				err := cmd.ICMPConnectivityCheck(masterPod, ip4Worker0NodeAddr, interfaceNameNet1)
+				Expect(err).ToNot(HaveOccurred(), "Failed to ping the worker nodes")
+
+				By("Verify ingress and egress TCP traffic over port 8888 between the external Pod and the test pods " +
+					"on the workers")
+				verifyTCPPortBeforeCustomFirewallActive(masterPod, testPodWorker0, ip4Worker0NodeAddr, portNum8888)
+
 				By("Define and create a NFTables custom rule blocking ingress TCP port 8888")
 				createMCAndWaitforMCPStable(tsparams.CustomFirewallIngressPort8888, mcNftablesName)
 
-				By("Verify ingress TCP traffic is blocked over port 8888 between the master Pod and the test" +
-					" pods on the workers")
+				By("Verify ICMP connectivity between the external Pod and the test pods on the workers")
+				err = cmd.ICMPConnectivityCheck(masterPod, ip4Worker0NodeAddr, interfaceNameNet1)
+				Expect(err).ToNot(HaveOccurred(), "Failed to ping the worker nodes")
 
-				err = cmd.ValidateTCPTraffic(masterPod, ipv4NodeAddrList, interfaceNameNet1, frrconfig.ContainerName,
-					portNum8888)
-				Expect(err).To(HaveOccurred(),
-					"Successfully sent ingress TCP traffic over port 8888 to the worker nodes")
+				By("Verify ingress TCP traffic is blocked and egress traffic is not blocked over port 8888")
+				verifyIngressTCPTrafficAfterCustomFirewallActive(masterPod, testPodWorker0, ipv4NodeAddrList, portNum8888)
 
-				By("Verify egress TCP traffic over port 8888 between the test Pod on the worker0 and the master pod")
-				err = cmd.ValidateTCPTraffic(testPodWorker0, []string{masterPodIPv4Address},
-					interfaceNameBrEx, "", portNum8888)
-				Expect(err).ToNot(HaveOccurred(),
-					"Failed to send egress TCP traffic over port 8888 to the pod on the master node")
+				By("Reboot worker-0")
+				rebootNodeAndWaitForMcpStable()
+
+				By("Recreate a static route to the external Pod network on worker node after reboot")
+				// GetMetalLbVirIP using the metallb virtal IP address variable for test pod IP addresses.
+				ipv4SecurityIPList, err = NetConfig.GetMetalLbVirIP()
+				Expect(err).ToNot(HaveOccurred(), "Failed to retrieve the ipv4SecurityIPList")
+
+				routeMap, err = netenv.BuildRoutesMapWithSpecificRoutes(testPodList, cnfWorkerNodeList, ipv4SecurityIPList)
+				Expect(err).ToNot(HaveOccurred(), "Failed to create route map with specific routes")
+
+				addDeleteStaticRouteOnWorkerNodes(testPodList, routeMap, "add", hubIPv4Network)
+
```

Comment:
You did it in beforeAll, why do you need it again?

Reply:
Because after the reboot the route is removed so it needs to be replaced